pygmt/accessors.py

```diff
@@ -45,9 +69,58 @@ class GMTDataArrayAccessor:
     >>> grid = xr.DataArray(
     ...     data, coords=[("latitude", lat), ("longitude", lon)]
     ... )
+    >>> # default to a gridline-registrated Cartesian grid
+    >>> grid.gmt.registration, grid.gmt.gtype
+    (0, 0)
     >>> # set it to a gridline-registered geographic grid
     >>> grid.gmt.registration = 0
     >>> grid.gmt.gtype = 1
+    >>> grid.gmt.registration, grid.gmt.gtype
+    (0, 1)
+
+    Note that the accessors are created once per :class:`xarray.DataArray`
+    instance, so you may lose these GMT-specific properties after manipulating
+    your grid.
+
+    Inplace assignment operators like ``*=`` don't create new instances, so the
+    properties are still kept:
+
+    >>> grid *= 2.0
+    >>> grid.gmt.registration, grid.gmt.gtype
+    (0, 1)
```

Comment:
Interesting, good to know that in-place assignment preserves the registration and gtype information!